lib/Classes.ts

```diff
@@ -0,0 +1 @@
+
```

Comment:
What is the purpose of this file?

Reply:
Was going to make a Class based API that leverages the functions, but it is a bit more involved than I anticipated. Perhaps it is a good thing to work on for 2.0. I will delete this file.